src/displayUI.js

```diff
@@ -0,0 +1,102 @@
+import { singletonInstance } from './mainClass';
+
+singletonInstance;
+
+// Display one item (element)
+export function displayOneCategory(el, singletonInstance) {
```

Comment:
Don't call el something that is not element, like dom element, call it wit more appropriate descriptive name.